views/layouts/partials/header.tpl.php

```diff
@@ -1,14 +1,25 @@
-<html>
 
-<head>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
-</head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
```

Comment:
You already have js and css related to bootstrap in your templates, you should not insert them in this partial template again.
Also <meta> tag has no place here, it's for <head> section, and we already have that in main layouts.
Have a look in basic.tpl.php and sidebar_page.tpl.php